futures01/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "futures01"
+edition = "2018"
```

Comment:
`edition` key is only available starting in Rust 1.31. (AFAIK I think the motivation for this PR was to have 0.1 and 0.3 co-exist in a version below Rust 1.31.)

Reply:
*sigh* that... makes sense :)

Reply:
cc @shepmaster I'm not sure how to do macro-reexporting on old editions. @dtolnay do you have any tricks here?

Reply:
A glob re-export (`pub use dependency::*`) will include macros back to rustc 1.15.0 on any edition.

Reply:
Ah, wonderful-- thank you!

Reply:
(seems incredibly surprising to me that `use dependency::*;` wouldn't actually bring the macro into scope, but `pub use dependency::*;` would reexport it :/ )

Reply:
Indeed. IIRC it was stabilized by accident.